api.md

```diff
@@ -25,6 +25,24 @@
 > - [show_tablespaces](#show_tablespaces)
 > - [time_bucket](#time_bucket)
 
+## TimescaleDB informational views [](timescaledb_information)
+
+## timescaledb_information.hypertable [](timescaledb_information.hypertable)
+
+Information about hypertables
```

Comment:
Can we match syntax of other helper functions? "Get information about hypertables."

Reply:
Also this should be a header (####) to match the format of other functions.  Maybe "Available columns" as the title?